client/i18n/locales/english/intro.json

```
@@ -3997,15 +4009,18 @@
       },
       "review-dynamic-programming": {
         "title": "Dynamic Programming Review",
-        "intro": [""]
+        "intro": [
+          "Before you're quizzed on dynamic programming, you should review what you've learned about it.",
+          "[Add more details]"
+        ]
       },
       "quiz-dynamic-programming": {
         "title": "Dynamic Programming Quiz",
         "intro": [""]
       },
       "review-python": {
         "title": "Python Review",
-        "intro": [""]
+        "intro": ["Review Python concepts to prepare for the upcoming quiz."]
```

this looks like a chapter review page instead of a quiz. so it should be updated.

```suggestion
        "intro": ["Review Python concepts to prepare for the upcoming exam."]
```

---

Also, as a sidenote, it looks like the placeholder for the python exam is missing. but that can added in later when we actually have an exam.